GSA/digitalgov.gov
# Add image sizes for improving performance

## Summary
Added `height`/`width` properties to certain images to avoid potential Cumulative Layout Shift (CLS) on page load.

### Preview
[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-improve-image-performance/)

### Solution
Chrome Lighthouse report flagged several self-hosted images that could cause CLS with no set `height`/`width`.
Hardcoded `height`/`width` on `img` to prevent CLS on page load.

### How To Test

1. Visit homepage
2. Run Lighthouse report and should see no digital.gov hosted layout images flagged for CLS.
3. AWS images will be flagged


**Before**
<img width="771" alt="Screen Shot 2023-07-17 at 2 27 16 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/98e56f26-1ca7-4e4b-8672-6ac7332e6253">


**After**
<img width="1678" alt="Screen Shot 2023-07-17 at 2 26 58 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/c3f1a1d4-7bff-40ff-94ea-e52407071689">

---

### Dev Checklist
